pkg/apis/virt/v1alpha1/types.go

```diff
@@ -67,7 +67,14 @@ type CPU struct {
 }
 
 type Memory struct {
-	Size *resource.Quantity `json:"size,omitempty"`
+	Size      *resource.Quantity `json:"size,omitempty"`
```

Comment:
Can we remove the pointer here, simply use the `resource.Quantity` type?

Reply:
Then how can we judge the zero value is assigned by user or the default? Maybe we can fill it when `size.IsZero()` at both cases.

Reply:
Zero is not a valid value, we can treat zero as empty here